src/sentry/api/endpoints/organization_index.py

```diff
@@ -222,23 +219,24 @@ def post(self, request: Request) -> Response:
                             team=team, organizationmember=om, is_active=True
                         )
 
-                    org_setup_complete.send_robust(
-                        instance=org, user=request.user, sender=self.__class__
-                    )
+                org_setup_complete.send_robust(
```

Comment:
This logic, which only exists in `getsentry`, works on subscription objects and doesn't belong to an outer transaction anyways, so this is more clarifying.